### ISSUE TITLE
Does RUC separatate train and test set?

### ISSUE THREAD

**Issue by shekkizh** [deu30303/RUC#6]
Hi,
I'm confused as to whether the reported performances are with train and test separated i.e training only with train dataset and evaluate using test dataset (I believe this is how SCAN reported results). In code, it seems like both the training dataset and test dataset are loaded with `train=True` meaning the model uses only the train dataset for training and reporting performance?!

Thank you for your time and for open-sourcing the code. ✌️

**Comment by deu30303** [comment]
Hi, thank you for your interest in our work!

For CIFAR-10 and CIFAR-20, train set and test set is not separated following previous papers (IIC, TSUC). 
For STL-10, we use 13,000 labeled images. 
In our paper, we report SCAN and our model's performance under same setting.
 
Thank you again for your interest.
